### PR TITLE
[CPDLP-3481] Scale up migration env

### PR DESCRIPTION
**Pull request by leandroalemao**
### Context

Ticket: [CPDLP-3481](https://dfedigital.atlassian.net/browse/CPDLP-3481)

The current migration environment is not scaled up and not performant when migrations run

### Changes proposed in this pull request

Scale up migration env.

[CPDLP-3481]: https://dfedigital.atlassian.net/browse/CPDLP-3481?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: terraform/application/config/migration.tfvars.json [hunk 1]
```diff
@@ -7,5 +7,9 @@
     "redis_cache_capacity": 0,
     "redis_cache_sku_name": "Basic",
     "enable_logit": true,
-    "worker_replicas": 5
+    "worker_replicas": 5,
+    "webapp_replicas": 4,
+    "webapp_memory_max": "4Gi",
+    "worker_memory_max": "2Gi",
+    "postgres_flexible_server_sku": "GP_Standard_D2ds_v4"
```
Comment (ethax-ross) [inline terraform/application/config/migration.tfvars.json:14]: We use `GP_Standard_D2ds_v5` in prod I think

File: terraform/application/config/migration.tfvars.json [hunk 2]
```diff
@@ -7,5 +7,9 @@
     "redis_cache_capacity": 0,
     "redis_cache_sku_name": "Basic",
     "enable_logit": true,
-    "worker_replicas": 5
+    "worker_replicas": 5,
+    "webapp_replicas": 4,
```
Comment (ethax-ross) [inline terraform/application/config/migration.tfvars.json:11]: Ditto as with the ECF PR; not sure we need to scale the webapps

Comment (peteryates) [reply terraform/application/config/migration.tfvars.json:11]: Is there any point in having more than 1 webapp?

File: terraform/application/config/migration.tfvars.json [hunk 3]
```diff
@@ -7,5 +7,7 @@
     "redis_cache_capacity": 0,
     "redis_cache_sku_name": "Basic",
     "enable_logit": true,
-    "worker_replicas": 5
```
Comment (cwrw) [inline terraform/application/config/migration.tfvars.json:10]: could we keep this please?

**Review by cwrw — APPROVED** [review]

**Review by ethax-ross — APPROVED** [review]

**Pull request merged**